src/pseudopeople/noise_functions.py

```diff
@@ -215,6 +216,7 @@ def _merge_dependents_and_guardians(
             if index_to_copy.empty:
                 continue
             noised_group_df = group_df.loc[index_to_copy]
+            noised_group_df["old_housing_type"] = noised_group_df["housing_type"]
```

Comment:
I don't love adding this column in the "real" data when it's only used for testing. But it also seems too big of a pain to refactor this somehow so that you can make a test fixture of the call and add the col there.

Instead, couldn't you get the "old_housing_type" from the unnoised_data in the tests?

Reply:
+1 if this is just for testing you could save a copy of the old_housing_type series and then map that to verify your test or something.